README.md

```diff
@@ -26,7 +26,7 @@ Site Milestone Dates\*: ~~June 30th~~✓ | August 31st | October 11th
 - Profile (links to blank page)
   --- Logout (links to blank page)
 - Support
-  --- Contact Us (links to blank page)
+  --- Contact Us (links to skeleton)
```

Comment:
Good catch. Is there a Contact Us page in the mockups? I can't find it, but I'm looking for something labeled "Contact Us"

Reply:
I've been following page 32 out of 125 [here](https://xd.adobe.com/view/6f6af599-8eb6-46b5-8731-e4c09ddd587e-3a39/screen/8875368e-2396-45c9-8ff1-b53624f6270f/) from the mockups